caffeine/build.gradle

```diff
@@ -55,6 +58,14 @@ dependencies {
   javaPoetCompile libraries.commonsLang3
 }
 
+task compileNoDebug(type: JavaCompile) {
+  source = sourceSets.gen.java.srcDirs
+  destinationDir = compileJava.destinationDir
+  options.debug = false
+  options.incremental = false
```

Comment:
I'm reworking this so that the compilation process doesn't overwrite the files (causes cache busting, harder to detect malicious supply chain attacks). By chance do you remember why `incremental` was false?